sources/engine/Stride.BepuPhysics/Stride.BepuPhysics.Navigation/Definitions/PathfindingSettings.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net)
+//  Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Stride.Core;
+
+namespace Stride.BepuPhysics.Navigation.Definitions;
+[DataContract(nameof(PathfindingSettings))]
```

Comment:
I think you should omit the string here, you should specify a string for your data contract only when you:
- Set a constant string literal (not `nameof(x)`) to keep serialization working with previous versions if the class is renamed. (although `DataAlias` could save your butt)
- Want to shorten the identifier used when this class is serialized in a yaml file

There's a couple of classes that have the same issue, feel free to add it to your todo list

Reply:
Yeah, this was intentional due to the second point you brought up. It has saved me from worrying about moving files around in a project since it doesnt care about the namespace but this is likely too vague a name and I do this causing issues. Will look into this soon.